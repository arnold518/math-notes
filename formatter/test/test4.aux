\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {1}{1}}}
\@writefile{toc}{\beamer@sectionintoc {1}{§ 1. Optimization Problem}{2}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {1}{1}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {1}{1}}}
\@writefile{nav}{\headcommand {\sectionentry {1}{§ 1. Optimization Problem}{2}{§ 1. Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{1}{2/2}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {2}{2}}}
\@writefile{toc}{\beamer@subsectionintoc {1}{1}{Definition of Optimization Problem}{3}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {2}{2}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{1}{1}{3}{Definition of Optimization Problem}}}
\@writefile{nav}{\headcommand {\slideentry {1}{1}{1}{3/3}{Definition of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {3}{3}}}
\newlabel{definition:1.1}{{4}{4}{Definition of Optimization Problem}{definition:1.1}{}}
\@writefile{snm}{\beamer@slide {definition:1.1}{4}}
\@writefile{nav}{\headcommand {\slideentry {1}{1}{2}{4/4}{Definition of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {4}{4}}}
\@writefile{nav}{\headcommand {\slideentry {1}{1}{3}{5/5}{Definition of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {5}{5}}}
\newlabel{definition:1.2}{{6}{6}{Definition of Optimization Problem}{definition:1.2}{}}
\@writefile{snm}{\beamer@slide {definition:1.2}{6}}
\@writefile{nav}{\headcommand {\slideentry {1}{1}{4}{6/6}{Definition of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {6}{6}}}
\newlabel{definition:1.3}{{7}{7}{Definition of Optimization Problem}{definition:1.3}{}}
\@writefile{snm}{\beamer@slide {definition:1.3}{7}}
\@writefile{nav}{\headcommand {\slideentry {1}{1}{5}{7/7}{Definition of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {7}{7}}}
\@writefile{toc}{\beamer@subsectionintoc {1}{2}{Examples of Optimization Problem}{8}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {3}{7}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{1}{2}{8}{Examples of Optimization Problem}}}
\@writefile{nav}{\headcommand {\slideentry {1}{2}{1}{8/8}{Examples of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {8}{8}}}
\newlabel{example:1.4}{{9}{9}{Examples of Optimization Problem}{example:1.4}{}}
\@writefile{snm}{\beamer@slide {example:1.4}{9}}
\@writefile{nav}{\headcommand {\slideentry {1}{2}{2}{9/9}{Examples of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{9}}}
\newlabel{example:1.5}{{10}{10}{Examples of Optimization Problem}{example:1.5}{}}
\@writefile{snm}{\beamer@slide {example:1.5}{10}}
\@writefile{nav}{\headcommand {\slideentry {1}{2}{3}{10/10}{Examples of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {10}{10}}}
\@writefile{nav}{\headcommand {\slideentry {1}{2}{3}{10/11}{Examples of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {10}{11}}}
\newlabel{concept:1.6}{{12}{12}{Examples of Optimization Problem}{concept:1.6}{}}
\@writefile{snm}{\beamer@slide {concept:1.6}{12}}
\@writefile{nav}{\headcommand {\slideentry {1}{2}{4}{12/12}{Examples of Optimization Problem}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {12}{12}}}
\@writefile{toc}{\beamer@subsectionintoc {1}{3}{Local and Global Minimum}{13}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {8}{12}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{1}{3}{13}{Local and Global Minimum}}}
\@writefile{nav}{\headcommand {\slideentry {1}{3}{1}{13/13}{Local and Global Minimum}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {13}{13}}}
\newlabel{definition:1.7}{{14}{14}{Local and Global Minimum}{definition:1.7}{}}
\@writefile{snm}{\beamer@slide {definition:1.7}{14}}
\@writefile{nav}{\headcommand {\slideentry {1}{3}{2}{14/14}{Local and Global Minimum}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {14}{14}}}
\@writefile{nav}{\headcommand {\slideentry {1}{3}{2}{14/15}{Local and Global Minimum}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {14}{15}}}
\@writefile{toc}{\beamer@sectionintoc {2}{§ 11. Variational Autoencoders}{16}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {2}{15}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {13}{15}}}
\@writefile{nav}{\headcommand {\sectionentry {2}{§ 11. Variational Autoencoders}{16}{§ 11. Variational Autoencoders}{0}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{1}{16/16}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {16}{16}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{2}{17/17}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {17}{17}}}
\newlabel{concept:11.1}{{18}{18}{§ 11. Variational Autoencoders}{concept:11.1}{}}
\@writefile{snm}{\beamer@slide {concept:11.1}{18}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{3}{18/18}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {18}{18}}}
\newlabel{concept:11.2}{{19}{19}{§ 11. Variational Autoencoders}{concept:11.2}{}}
\@writefile{snm}{\beamer@slide {concept:11.2}{19}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{4}{19/19}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {19}{19}}}
\newlabel{concept:11.3}{{20}{20}{§ 11. Variational Autoencoders}{concept:11.3}{}}
\@writefile{snm}{\beamer@slide {concept:11.3}{20}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{5}{20/20}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {20}{20}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{6}{21/21}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {21}{21}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{6}{21/22}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {21}{22}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{1}{Latent Variable Model}{23}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {16}{22}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{1}{23}{Latent Variable Model}}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{1}{23/23}{Latent Variable Model}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {23}{23}}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{2}{24/24}{Latent Variable Model}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {24}{24}}}
\newlabel{definition:11.4}{{25}{25}{Latent Variable Model}{definition:11.4}{}}
\@writefile{snm}{\beamer@slide {definition:11.4}{25}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{3}{25/25}{Latent Variable Model}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {25}{25}}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{3}{25/26}{Latent Variable Model}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {25}{26}}}
\newlabel{example:11.5}{{27}{27}{Latent Variable Model}{example:11.5}{}}
\@writefile{snm}{\beamer@slide {example:11.5}{27}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{4}{27/27}{Latent Variable Model}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {27}{27}}}
\@writefile{nav}{\headcommand {\slideentry {2}{1}{4}{27/28}{Latent Variable Model}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {27}{28}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{2}{Training Latent Variable Model with Importance Sampling}{29}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {23}{28}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{2}{29}{Training Latent Variable Model with Importance Sampling}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{1}{29/29}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {29}{29}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{2}{30/30}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {30}{30}}}
\newlabel{concept:11.6}{{31}{31}{Training Latent Variable Model with Importance Sampling}{concept:11.6}{}}
\@writefile{snm}{\beamer@slide {concept:11.6}{31}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{3}{31/31}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {31}{31}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{3}{31/32}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {31}{32}}}
\newlabel{concept:11.7}{{33}{33}{Training Latent Variable Model with Importance Sampling}{concept:11.7}{}}
\@writefile{snm}{\beamer@slide {concept:11.7}{33}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{4}{33/33}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {33}{33}}}
\newlabel{concept:11.8}{{34}{34}{Training Latent Variable Model with Importance Sampling}{concept:11.8}{}}
\@writefile{snm}{\beamer@slide {concept:11.8}{34}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{5}{34/34}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {34}{34}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{6}{35/35}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {35}{35}}}
\newlabel{concept:11.9}{{36}{36}{Training Latent Variable Model with Importance Sampling}{concept:11.9}{}}
\@writefile{snm}{\beamer@slide {concept:11.9}{36}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{7}{36/36}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {36}{36}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{7}{36/37}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {36}{37}}}
\newlabel{concept:11.10}{{38}{38}{Training Latent Variable Model with Importance Sampling}{concept:11.10}{}}
\@writefile{snm}{\beamer@slide {concept:11.10}{38}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{8}{38/38}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {38}{38}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{8}{38/39}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {38}{39}}}
\newlabel{concept:11.11}{{40}{40}{Training Latent Variable Model with Importance Sampling}{concept:11.11}{}}
\@writefile{snm}{\beamer@slide {concept:11.11}{40}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{9}{40/40}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {40}{40}}}
\newlabel{concept:11.12}{{41}{41}{Training Latent Variable Model with Importance Sampling}{concept:11.12}{}}
\@writefile{snm}{\beamer@slide {concept:11.12}{41}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{10}{41/41}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {41}{41}}}
\@writefile{nav}{\headcommand {\slideentry {2}{2}{10}{41/42}{Training Latent Variable Model with Importance Sampling}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {41}{42}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{3}{Definition of VAE}{43}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {29}{42}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{3}{43}{Definition of VAE}}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{1}{43/43}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {43}{43}}}
\newlabel{definition:11.13}{{44}{44}{Definition of VAE}{definition:11.13}{}}
\@writefile{snm}{\beamer@slide {definition:11.13}{44}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{2}{44/44}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {44}{44}}}
\newlabel{concept:11.14}{{45}{45}{Definition of VAE}{concept:11.14}{}}
\@writefile{snm}{\beamer@slide {concept:11.14}{45}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{3}{45/45}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {45}{45}}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{3}{45/46}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {45}{46}}}
\newlabel{concept:11.15}{{47}{47}{Definition of VAE}{concept:11.15}{}}
\@writefile{snm}{\beamer@slide {concept:11.15}{47}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{4}{47/47}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {47}{47}}}
\newlabel{definition:11.16}{{48}{48}{Definition of VAE}{definition:11.16}{}}
\@writefile{snm}{\beamer@slide {definition:11.16}{48}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{5}{48/48}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {48}{48}}}
\@writefile{nav}{\headcommand {\slideentry {2}{3}{5}{48/49}{Definition of VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {48}{49}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{4}{VAE Standard Instance}{50}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {43}{49}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{4}{50}{VAE Standard Instance}}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{1}{50/50}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {50}{50}}}
\newlabel{definition:11.17}{{51}{51}{VAE Standard Instance}{definition:11.17}{}}
\@writefile{snm}{\beamer@slide {definition:11.17}{51}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{2}{51/51}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {51}{51}}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{2}{51/52}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {51}{52}}}
\newlabel{concept:11.18}{{53}{53}{VAE Standard Instance}{concept:11.18}{}}
\@writefile{snm}{\beamer@slide {concept:11.18}{53}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{3}{53/53}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {53}{53}}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{3}{53/54}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {53}{54}}}
\newlabel{concept:11.19}{{55}{55}{VAE Standard Instance}{concept:11.19}{}}
\@writefile{snm}{\beamer@slide {concept:11.19}{55}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{4}{55/55}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {55}{55}}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{4}{55/56}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {55}{56}}}
\newlabel{concept:11.20}{{57}{57}{VAE Standard Instance}{concept:11.20}{}}
\@writefile{snm}{\beamer@slide {concept:11.20}{57}}
\@writefile{nav}{\headcommand {\slideentry {2}{4}{5}{57/57}{VAE Standard Instance}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {57}{57}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{5}{Training VAE}{58}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {50}{57}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{5}{58}{Training VAE}}}
\@writefile{nav}{\headcommand {\slideentry {2}{5}{1}{58/58}{Training VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {58}{58}}}
\newlabel{concept:11.21}{{59}{59}{Training VAE}{concept:11.21}{}}
\@writefile{snm}{\beamer@slide {concept:11.21}{59}}
\@writefile{nav}{\headcommand {\slideentry {2}{5}{2}{59/59}{Training VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {59}{59}}}
\newlabel{concept:11.22}{{60}{60}{Training VAE}{concept:11.22}{}}
\@writefile{snm}{\beamer@slide {concept:11.22}{60}}
\@writefile{nav}{\headcommand {\slideentry {2}{5}{3}{60/60}{Training VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {60}{60}}}
\@writefile{nav}{\headcommand {\slideentry {2}{5}{3}{60/61}{Training VAE}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {60}{61}}}
\@writefile{toc}{\beamer@subsectionintoc {2}{6}{Researches}{62}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {58}{61}}}
\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{6}{62}{Researches}}}
\@writefile{nav}{\headcommand {\slideentry {2}{6}{1}{62/62}{Researches}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {62}{62}}}
\newlabel{concept:11.23}{{63}{63}{Researches}{concept:11.23}{}}
\@writefile{snm}{\beamer@slide {concept:11.23}{63}}
\@writefile{nav}{\headcommand {\slideentry {2}{6}{2}{63/63}{Researches}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {63}{63}}}
\@writefile{nav}{\headcommand {\slideentry {2}{6}{2}{63/64}{Researches}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {63}{64}}}
\newlabel{concept:11.24}{{65}{65}{Researches}{concept:11.24}{}}
\@writefile{snm}{\beamer@slide {concept:11.24}{65}}
\@writefile{nav}{\headcommand {\slideentry {2}{6}{3}{65/65}{Researches}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {65}{65}}}
\@writefile{nav}{\headcommand {\slideentry {2}{6}{3}{65/66}{Researches}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {65}{66}}}
\newlabel{concept:11.25}{{67}{67}{Researches}{concept:11.25}{}}
\@writefile{snm}{\beamer@slide {concept:11.25}{67}}
\@writefile{nav}{\headcommand {\slideentry {2}{6}{4}{67/67}{Researches}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {67}{67}}}
\@writefile{nav}{\headcommand {\beamer@partpages {1}{67}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {62}{67}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {16}{67}}}
\@writefile{nav}{\headcommand {\beamer@documentpages {67}}}
\@writefile{nav}{\headcommand {\gdef \inserttotalframenumber {67}}}
\gdef \@abspage@last{67}
