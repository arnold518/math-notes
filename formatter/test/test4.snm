\beamer@slide {definition:1.1}{4}
\beamer@slide {definition:1.2}{6}
\beamer@slide {definition:1.3}{7}
\beamer@slide {example:1.4}{9}
\beamer@slide {example:1.5}{10}
\beamer@slide {concept:1.6}{12}
\beamer@slide {definition:1.7}{14}
\beamer@slide {concept:11.1}{18}
\beamer@slide {concept:11.2}{19}
\beamer@slide {concept:11.3}{20}
\beamer@slide {definition:11.4}{25}
\beamer@slide {example:11.5}{27}
\beamer@slide {concept:11.6}{31}
\beamer@slide {concept:11.7}{33}
\beamer@slide {concept:11.8}{34}
\beamer@slide {concept:11.9}{36}
\beamer@slide {concept:11.10}{38}
\beamer@slide {concept:11.11}{40}
\beamer@slide {concept:11.12}{41}
\beamer@slide {definition:11.13}{44}
\beamer@slide {concept:11.14}{45}
\beamer@slide {concept:11.15}{47}
\beamer@slide {definition:11.16}{48}
\beamer@slide {definition:11.17}{51}
\beamer@slide {concept:11.18}{53}
\beamer@slide {concept:11.19}{55}
\beamer@slide {concept:11.20}{57}
\beamer@slide {concept:11.21}{59}
\beamer@slide {concept:11.22}{60}
\beamer@slide {concept:11.23}{63}
\beamer@slide {concept:11.24}{65}
\beamer@slide {concept:11.25}{67}
