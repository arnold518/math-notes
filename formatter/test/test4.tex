\documentclass{beamer}
\makeatletter
\@ifclassloaded{beamer}{\usecolortheme{orchid}}{\usepackage{beamerarticle}}
\makeatother

\NewDocumentEnvironment{preuve}{O{Preuve}}{\begin{block}{#1}}{\end{block}}

\begin{document}
\begin{frame}
\begin{alertblock}{Berge, 1957}
Un couplage $C$ est de cardinal maximal si et seulement s'il n'admet pas de chemin augmentant.
\end{alertblock}
\pause
\begin{preuve}[Preuve du sens non couvert par le lemme]
some already long text
\mergeFrame{preuve}{block}
\end{preuve}
\end{frame}

\begin{frame}
\begin{block}{}

Soit $p$ un chemin maximal dans $K$ : \uncover<2->{sa première et sa dernière arêtes sont dans $C_2$.}
\end{block}

Remarque : ce lemme et ce théorème n'utilisent pas le fait que le graphe est biparti.\\
Cette propriété va cependant permettre de faciliter la recherche d'un chemin augmentant.
\end{frame}
\end{document}