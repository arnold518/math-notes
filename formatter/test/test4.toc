\beamer@sectionintoc {1}{ยง 1. Optimization Problem}{2}{0}{1}
\beamer@subsectionintoc {1}{1}{Definition of Optimization Problem}{3}{0}{1}
\beamer@subsectionintoc {1}{2}{Examples of Optimization Problem}{8}{0}{1}
\beamer@subsectionintoc {1}{3}{Local and Global Minimum}{13}{0}{1}
\beamer@sectionintoc {2}{ยง 11. Variational Autoencoders}{16}{0}{2}
\beamer@subsectionintoc {2}{1}{Latent Variable Model}{23}{0}{2}
\beamer@subsectionintoc {2}{2}{Training Latent Variable Model with Importance Sampling}{29}{0}{2}
\beamer@subsectionintoc {2}{3}{Definition of VAE}{43}{0}{2}
\beamer@subsectionintoc {2}{4}{VAE Standard Instance}{50}{0}{2}
\beamer@subsectionintoc {2}{5}{Training VAE}{58}{0}{2}
\beamer@subsectionintoc {2}{6}{Researches}{62}{0}{2}
