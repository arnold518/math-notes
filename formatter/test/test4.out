\BOOKMARK [2][]{Outline0.1}{\376\377\000\247\000\040\0001\000.\000\040\000O\000p\000t\000i\000m\000i\000z\000a\000t\000i\000o\000n\000\040\000P\000r\000o\000b\000l\000e\000m}{}% 1
\BOOKMARK [3][]{Outline0.1.1.3}{\376\377\000D\000e\000f\000i\000n\000i\000t\000i\000o\000n\000\040\000o\000f\000\040\000O\000p\000t\000i\000m\000i\000z\000a\000t\000i\000o\000n\000\040\000P\000r\000o\000b\000l\000e\000m}{Outline0.1}% 2
\BOOKMARK [3][]{Outline0.1.2.8}{\376\377\000E\000x\000a\000m\000p\000l\000e\000s\000\040\000o\000f\000\040\000O\000p\000t\000i\000m\000i\000z\000a\000t\000i\000o\000n\000\040\000P\000r\000o\000b\000l\000e\000m}{Outline0.1}% 3
\BOOKMARK [3][]{Outline0.1.3.13}{\376\377\000L\000o\000c\000a\000l\000\040\000a\000n\000d\000\040\000G\000l\000o\000b\000a\000l\000\040\000M\000i\000n\000i\000m\000u\000m}{Outline0.1}% 4
\BOOKMARK [2][]{Outline0.2}{\376\377\000\247\000\040\0001\0001\000.\000\040\000V\000a\000r\000i\000a\000t\000i\000o\000n\000a\000l\000\040\000A\000u\000t\000o\000e\000n\000c\000o\000d\000e\000r\000s}{}% 5
\BOOKMARK [3][]{Outline0.2.1.23}{\376\377\000L\000a\000t\000e\000n\000t\000\040\000V\000a\000r\000i\000a\000b\000l\000e\000\040\000M\000o\000d\000e\000l}{Outline0.2}% 6
\BOOKMARK [3][]{Outline0.2.2.29}{\376\377\000T\000r\000a\000i\000n\000i\000n\000g\000\040\000L\000a\000t\000e\000n\000t\000\040\000V\000a\000r\000i\000a\000b\000l\000e\000\040\000M\000o\000d\000e\000l\000\040\000w\000i\000t\000h\000\040\000I\000m\000p\000o\000r\000t\000a\000n\000c\000e\000\040\000S\000a\000m\000p\000l\000i\000n\000g}{Outline0.2}% 7
\BOOKMARK [3][]{Outline0.2.3.43}{\376\377\000D\000e\000f\000i\000n\000i\000t\000i\000o\000n\000\040\000o\000f\000\040\000V\000A\000E}{Outline0.2}% 8
\BOOKMARK [3][]{Outline0.2.4.50}{\376\377\000V\000A\000E\000\040\000S\000t\000a\000n\000d\000a\000r\000d\000\040\000I\000n\000s\000t\000a\000n\000c\000e}{Outline0.2}% 9
\BOOKMARK [3][]{Outline0.2.5.58}{\376\377\000T\000r\000a\000i\000n\000i\000n\000g\000\040\000V\000A\000E}{Outline0.2}% 10
\BOOKMARK [3][]{Outline0.2.6.62}{\376\377\000R\000e\000s\000e\000a\000r\000c\000h\000e\000s}{Outline0.2}% 11
